code/game/objects/effects/spawners/bundle.dm

```diff
@@ -63,6 +63,15 @@
 		/obj/item/clothing/under/suit/black,
 		/obj/item/clothing/head/that)
 
+/obj/effect/spawner/bundle/costume/referee
+	name = "referee costume spawner"
+	items = list(
+		/obj/item/clothing/mask/whistle,
+		/obj/item/clothing/gloves/color/latex,
```

Comment:
```suggestion
		/obj/item/clothing/gloves/color/white,
```

Reply:
Originally I had white gloves but I thought latex was better suited since combat referee's do have to deal with blood.  Aesthetics wise, they are both white colored.